grails-app/views/project/index.gsp

```diff
@@ -70,7 +70,22 @@
                         <a href="${createLink(controller: 'transcribe', action: 'index', id: projectInstance.id)}" class="btn btn-primary btn-lg" role="button">Get Started <span class="glyphicon glyphicon-arrow-right"></span></a>
                         <g:if test="${projectInstance.tutorialLinks}">
```

Comment:
Should this test be if tutorialLinks OR projectInstance.tutorials?  Or a message on the admin screen saying the tutorials won't be linked to the project unless you add something to the tutorial information/links section?